packages/components/addon/components/hds/pagination-bar/index.hbs

```diff
@@ -1,70 +1,31 @@
-<div class={{this.classNames}} ...attributes>
-  {{#if this.showTotalCount}}
-    <Hds::PaginationBar::PaginationTotalCount @hasTotalItems={{this.hasTotalItems}} />
-  {{/if}}
-
-  <Hds::PaginationBar::Pagination @type={{this.type}} />
-
-  {{#if this.showPageSize}}
-    <Hds::PaginationBar::PaginationPageSize />
-  {{/if}}
-</div>
-
-{{!
-  TEMP ROUGH NOTES
-  ------------------
-  Figuring out class naming and DOM structure...
-
-  WIP Structure (some names represent sub-components, others just represent HTML/CSS structure):
-    * PaginationBar
-      * PaginationTotalCount (optional)
-
-      * Pagination
-        * page-prev
-        * pages
-          * page
-            * page-link (some content aren't links though...)
-        * page-next
-
-      * PaginationPageSize (optional)
-
-APIs
-------
-PaginationBar:
-  X type=
-    "numbered" (default)
-    "compact"
-  X showTotalCount ?) =
-    true (default)
-    false
-  X showPageSize =
-    true (default)
-    false
-  isTruncated =
-    true (default ?)
-    false
-  X hasTotalItems=
-    true (default)
-    false
-
-TotalCount:
-  X hasTotalItems= (take values passed down from PaginationBar unless used as stand alone component)
-    true (default)
-    false
-
-Pagination:
-  X type= (take values passed down from PaginationBar unless used as stand alone component)
-    "numbered" (default)
-    "compact"
-  isTruncated= (take values passed down from PaginationBar unless used as stand alone component)
-    true (default ?)
-    false
-  * PageNumber:
-      isSelected (NOT isCurrent) (for page number, not exposed)
-      Truncated/ellipsis ? (for page number, not exposed)
-  * Prev/Next buttons:
-    * isDisabled (not exposed)
-
-PageSize:
-  ?
-}}
\ No newline at end of file
+<div class="hds-pagination-bar" ...attributes>
+  {{yield
+    (hash
+      TotalCount=(component
+        "hds/pagination-bar/total-count"
+        hasTotalItems=@hasTotalItems
```

Comment:
you don't need a specific `hasTotalItems` argument, you can rely on the existence (not `undefined` of `@totalItems` to know if it has total items or not).

Reply:
My understanding of whether to show the total number of items (hasTotalItems) was that it was just an option for the end user to customize the visual UI. 

For example, displaying "1–10" vs. "1–10 of 100". By default it seems it should be true.

Maybe the name is confusing though and we could come up with a clearer name?

Reply:
I agree it's confusing. I will work more on the documentation after this.

Reply:
Resolving the conversation for now, we can review this later.